src/core-styling/mixins/_scale.scss

```diff
@@ -0,0 +1,6 @@
+@use "../settings/font";
+@use "../../../node_modules/modularscale-sass/stylesheets/modularscale";
```

Comment:
Not that happy about reaching down directly into `node_modules`, but equally not happy about tying the sass implementation to Webpack by [using the `~` prefix](https://webpack.js.org/loaders/sass-loader/#resolving-import-at-rules) either 🤔.